src/core/Statement.ml

```diff
@@ -3,6 +3,15 @@
 
 (** {1 Statement} *)
 
+module OptionSet = Set.Make(
+   struct
+      let compare x y = Pervasives.compare x y
```

Comment:
Don't understand this.

Reply:
I mean, above, there's something about `let compare = CCOrd.(option int)` in another comment. And trivially useful modules like that can become `Util.IntOptionSet` instead of being created in several places.